src/layer.js

```diff
@@ -107,9 +125,34 @@ function sigmoid(value) {
 
 /**
  * Function that calculates the derivate of the sigmoid function.
+ * given the value of the sigmoid function at that point
  * @param value
  * @returns {number}
  */
 function sigmoidGradient(value) {
     return value * (1 - value);
 }
+
+/**
+ * Function that caclulates the hyperbolic tangent (tanh) function at some value
+ * @param value
+ * @returns {number}
+**/
+
+function tanh(value){
+    return Math.tanh(value);
+}
+
+/**
+ * Function that caclulates the derivative of 
+ * hyperbolic tangent (tanh) function given the value
+ * of the hyperbolic tangent function at that point
+ * @param value
+ * @returns {number}
+**/
+
+function tanhGradient(value){
+    return 1-Math.pow(value, 2)
```

Comment:
When I installed the cheminfo/eslint-config I found 96 errors mostly in lines that I did not write. 

I attempted to fix some of these issues in: dc279fa73083b1fc01bbdf7bbb84975d0875125c